tests/serializers.py

```diff
@@ -170,6 +171,10 @@ class Meta:
         requires=['location.cat_set.*'],
         deferred=True
     )
+    number_of_katz = DynamicMethodField(
+        requires=['location__cat_set'],
```

Comment:
Hmm, so this seems like it should be equivalent to `requires=['location.cat_set']`. Is that right? Does this issue persist if we use the form: `requires=['location.cat_set.*']`?